### ISSUE TITLE
请问怎么监控多个region的各实例

### ISSUE THREAD

**Issue by rambolijian** [aylei/aliyun-exporter#6]

**Comment by aylei** [comment]
启动多个 aliyun-exporter 实例

**Comment by rambolijian** [comment]
以端口区分，然后配置一个prometheus，grafana去收集和展示对吗

**Comment by aylei** [comment]
可以用端口区分, k8s 或 docker-compose 也可以直接起多个不同的 aliyun-exporter 服务

**Comment by rambolijian** [comment]
多谢，docker-compose起多个，不就是多套prometheus嘛，一个region对应一个prometheus了，是这么理解嘛

**Comment by aylei** [comment]
不是, 仅仅是多定义几个 aliyun-exporter, 比如 aliyun-exporter1, aliyun-exporter2

可以参考 tidb-compose 的例子: https://github.com/pingcap/tidb-docker-compose/blob/master/docker-compose.yml

**Comment by rambolijian** [comment]
多谢eylei，想请问下docker-compose/exporter/aliyun-exporter.yml中这么多metrics，随便自己取名的？还是和哪里关联的，我知道有这么多metrics，谢谢

**Comment by aylei** [comment]
参考 README 中的 Metrics Meta 一节:

## Metrics Meta

`aliyun-exporter` shipped with a simple site hosting the metrics meta from the CloudMonitor API. You can visit the metric meta in [localhost:9525](http://localhost:9525) after launching the exporter.

* `host:port` will host all the available monitor projects
* `host:port/projects/{project}` will host the metrics meta of a certain project
* `host:port/yaml/{project}` will host a config YAML of the project's metrics

you can easily navigate in this pages by hyperlink.
